frontend/src/screens/subwallets/SubwalletCreated.tsx

```diff
@@ -0,0 +1,377 @@
+import {
+  AlertCircle,
+  AlertTriangleIcon,
+  CopyIcon,
+  ExternalLinkIcon,
+  TriangleAlert,
+} from "lucide-react";
+import QRCode from "react-qr-code";
+import { Link, useLocation, useNavigate } from "react-router-dom";
+import AppHeader from "src/components/AppHeader";
+import ExternalLink from "src/components/ExternalLink";
+import { AppleIcon } from "src/components/icons/Apple";
+import { PlayStoreIcon } from "src/components/icons/PlayStore";
+import { ZapStoreIcon } from "src/components/icons/ZapStore";
+import { IsolatedAppTopupDialog } from "src/components/IsolatedAppTopupDialog";
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "src/components/ui/accordion";
+import { Alert, AlertDescription, AlertTitle } from "src/components/ui/alert";
+import { Badge } from "src/components/ui/badge";
+import { Button, ExternalLinkButton } from "src/components/ui/button";
+import {
+  Card,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "src/components/ui/card";
+import { Input } from "src/components/ui/input";
+import { Label } from "src/components/ui/label";
+import {
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+} from "src/components/ui/popover";
+import { Textarea } from "src/components/ui/textarea";
+import { useToast } from "src/components/ui/use-toast";
+import { useApp } from "src/hooks/useApp";
+import { useNodeConnectionInfo } from "src/hooks/useNodeConnectionInfo";
+import { copyToClipboard } from "src/lib/clipboard";
+import { ConnectAppCard } from "src/screens/apps/AppCreated";
+import { CreateAppResponse } from "src/types";
+
+export function SubwalletCreated() {
+  const { data: nodeConnectionInfo } = useNodeConnectionInfo();
+  const { toast } = useToast();
+
+  const { state } = useLocation();
+  const navigate = useNavigate();
+  const createAppResponse = state as CreateAppResponse;
+  const { data: app } = useApp(createAppResponse.pairingPublicKey, true);
+
+  if (!createAppResponse?.pairingUri) {
+    navigate("/");
+    return null;
+  }
+
+  const name = createAppResponse.name;
+  const appPublicKey = createAppResponse.pairingPublicKey;
+  const connectionSecret = createAppResponse.pairingSecretKey;
```

Comment:
I think this should be `pairingUri`
```suggestion
  const connectionSecret = createAppResponse.pairingUri;
```